civihr_employee_portal/features/civihr_default_permissions/civihr_default_permissions.info

```diff
@@ -266,6 +266,7 @@ features[user_permission][] = view revisions
 features[user_permission][] = view staff directory
 features[user_permission][] = view the administration theme
 features[user_permission][] = view vacancies
+features[user_permission][] = access CiviCRM developer menu and tools
```

Comment:
The permissions are listed here in alphabetical order. Can you please add this right after the "access CiviCRM"?